bitcoin-units/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "bitcoin-units"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>", "The Rust Bitcoin developers"]
+license = "CC0-1.0"
+homepage = "https://github.com/rust-bitcoin/rust-bitcoin/"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
+documentation = "https://docs.rs/bitcoin/"
```

Comment:
The link is wrong and a good demonstration why omitting it is better :)

Reply:
Fixed to be the correct URL once the crate is released

Reply:
Why not omit it? (crates.io will still add a link to docs.rs).

Reply:
Sweet, I did not know that, will remove.